propagation/b3.go

```diff
@@ -25,7 +25,7 @@ func MarshalB3TraceContext(ctx context.Context, prop *PropagationContext) (conte
 		return ctx, headerMap
 	}
 	ctx = trace.ContextWithSpan(ctx, otelSpan)
-	propagator := b3.B3{}
+	propagator := b3.B3{InjectEncoding: b3.B3MultipleHeader}
```

Comment:
OTEL changed default behavior when no encoding provided to create a single B3 header

Reply:
Do I understand this correctly? 

Previously the `B3MultipleHeader` would be used by default, but now the default is an `B3Unspecified`? So we're injecting explicitly the previous default?

Reply:
If unspecified, it used to make it a multi-header. Now if unspecified, it makes it a single-header.

Reply:
OK. So this is us saying multi-header all the time?

Reply:
Correct -- that's keeping it in sync with our existing API. Without this change we would inherit the "new default" which would be a breaking change for our users.

Reply:
👍  Cool. Thank you for the explanation!